flybywiresim/a32nx
# fix(mcdu): check to data

## Summary of Changes

- `CHECK TO DATA` message appears when either v1/vr/v2 has been entered and FLAPS or THS or FLEX TEMP is changed.
- Confirm prompt only appears when changing TO RUNWAY

## Screenshots (if necessary)


![image](https://cdn.discordapp.com/attachments/739223532269076561/812736183410753636/unknown.png)

## References

[Pilot Feedback](https://discord.com/channels/738864299392630914/754851312306487326/808681537834450946)
![image](https://cdn.discordapp.com/attachments/739223532269076561/812697859068788746/unknown.png)




## Additional context



Discord username (if different from GitHub): DerL30N#3751

## Testing instructions


Pre conditions:
- Take Off runway is defined prior to performance page entries:
   1. Define Take Off runway
   2. Insert V1/VR/V2/FLEX TMP
   3. Change SID
   4. no confirm prompt and no message is displayed
   5. Change Take Off runway
   6. Confirm prompt is visible  (V1/VR/V2/FLEX TMP needs to be confirmed) and message are displayed
- Take Off runway is not defined prior to performance page entries:
   1. Insert V1/VR/V2/FLEX TMP
   2. Insert Take Off runway
   3. Conform prompt is visible (V1/VR/V2 needs to be confirmed) and message is displayed

Changing FLEX TEMP and FLAPS/THS only brings up the message.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
